cmd/thanos/rule_test2.go

```diff
@@ -0,0 +1,68 @@
+package main
```

Comment:
WHy is this named `_test2.go` and not `_test.go` ?

Reply:
+1

Reply:
WUT! This should not go into this commit TBH, sorry!